kernel/src/main/java/org/kframework/compile/GatherVarsVisitor.java

```diff
@@ -14,6 +15,7 @@ public class GatherVarsVisitor extends RewriteAwareVisitor {
     private final Set<KVariable> vars;
     private final Set<KEMException> errors;
     private final boolean errorExistential;
+    private boolean isInExistsLhs = false;
```

Comment:
Why is this called `isInExistsLhs` instead of `isInMLBinder`?

Reply:
Fixed.